deeplabcut/pose_estimation_tensorflow/core/openvino/session.py

```diff
@@ -0,0 +1,131 @@
+import os
+import subprocess
+
+import numpy as np
+from tqdm import tqdm
+import cv2
+
+try:
+    from openvino.runtime import Core, AsyncInferQueue
+    is_openvino_available = True
+except ImportError:
+    is_openvino_available = False
+
+class OpenVINOSession:
+    def __init__(self, cfg, device):
+        self.core = Core()
+        self.xml_path = cfg["init_weights"] + ".xml"
+        self.device = device
+
+        # Convert a frozen graph to OpenVINO IR format
+        if not os.path.exists(self.xml_path):
+            subprocess.run(
+                [
+                    "mo",
+                    "--output_dir",
+                    os.path.dirname(cfg["init_weights"]),
+                    "--input_model",
+                    cfg["init_weights"] + ".pb",
+                    "--input_shape",
+                    "[1, 747, 832, 3]",
```

Comment:
Should image shape be hardcoded?

Reply:
No, this is just a default value. Later network is reshaped to actual image sizes (see `self.net.reshape`)

Reply:
Got it!